Sources/JavaKitMacros/JavaMethodMacro.swift

```diff
@@ -94,7 +94,8 @@ extension JavaMethodMacro: BodyMacro {
 
     return [
       """
-      self = \(raw: tryKeyword) Self.dynamicJavaNewObject(in: environment\(raw: arguments))
+      let _environment = environment == nil ? \(raw: tryKeyword) JavaVirtualMachine.shared().environment() : environment!
```

Comment:
maybe this would be nicer, without resorting to the ! 

```suggestion
      let environment = 
        if let environment { 
          environment
        } else {
          \(raw: tryKeyword) JavaVirtualMachine.shared().environment()
        }
```

Reply:
Ah, I had the same comment earlier ;)